internal/service/studies/contracts.go

```diff
@@ -133,3 +134,49 @@ func (s *Service) GetContract(ctx context.Context,
 	}
 	return s.s3.GetObject(ctx, metadata)
 }
+
+func (s *Service) UpdateContract(
+	ctx context.Context,
+	assetId uuid.UUID,
+	contractId uuid.UUID,
+	contractMetadata types.Contract,
+	pdfContractObj *types.S3Object,
+) error {
+	log.Debug().Any("contractId", contractId).Msg("Updating contract")
+
```

Comment:
will need to call https://github.com/ucl-arc-tre/portal/blob/4b84b9f4520f7675bc875708668792c3e9b63713/internal/service/studies/assets.go#L198
once merged